subprojects/parseq/src/main/java/com/linkedin/parseq/internal/PlanDeactivationListener.java

```diff
@@ -5,8 +5,6 @@
 /**
  * This listener interface allows receiving notifications about plan deactivation.
  * It can be used for monitoring and to optimize task implementation e.g. to implement batching.
- * See {@link EngineBuilder#setPlanActivityListener(PlanDeactivationListener)}
```

Comment:
Why do you remove this? This method is there in `EngineBuilder`.

Reply:
This method was replaced by `setPlanDeactivationListener` in https://github.com/linkedin/parseq/commit/bfcc5ffb2680fc36e9a7bf4f139f8df479bb2b4d but this comment never got changed. I wasn't able to find this method that's why I removed it but let me replace it with the new one